crates/tests-e2e/src/tracing.rs

```diff
@@ -0,0 +1,14 @@
+use std::sync::Once;
+
+pub fn init_tracing() {
+    static TRACING_TEST_SUBSCRIBER: Once = Once::new();
+
+    TRACING_TEST_SUBSCRIBER.call_once(|| {
```

Comment:
❓Do we need this to be a `static`? Would this even be shared between tests? I don't think that's how it works anyway.

Reply:
I realise that we already do this in `ln-dlc-node`, but I also don't understand it there.

Reply:
it needs to if more than one test will be built in one binary, because we should only initialise it once. 
the crucial part is the `Once`, not the static itself - it ensures that we can't accidentally init it more than once.